pkg/security/secl/rules/model.go

```diff
@@ -131,6 +129,88 @@ func (a *ActionDefinition) Name() ActionName {
 	}
 }
 
+// Equals returns true if the action is equal to another action
+func (a *ActionDefinition) Equals(other *ActionDefinition) bool {
```

Comment:
Can we use [reflect.DeepEqual](https://pkg.go.dev/reflect#DeepEqual) instead of doing the comparison ourselves?